splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/TargetUrlValidator.java

```diff
@@ -4,16 +4,24 @@
 import org.apache.camel.Message;
 import org.apache.camel.Processor;
 import org.apache.commons.validator.routines.UrlValidator;
+import org.apache.http.ProtocolException;
 
 public class TargetUrlValidator implements Processor {
 
     public void process(Exchange exchange) throws Exception {
-        String[] schemes = { "http", "https" };
-        UrlValidator urlValidator = new UrlValidator(schemes, UrlValidator.ALLOW_LOCAL_URLS);
+        String[] http_schemes = { "http" };
+        String[] https_schemes = { "https" };
+        UrlValidator httpUrlValidator = new UrlValidator(http_schemes, UrlValidator.ALLOW_LOCAL_URLS);
+        UrlValidator httpsUrlValidator = new UrlValidator(https_schemes, UrlValidator.ALLOW_LOCAL_URLS);
 
         Message in = exchange.getIn();
         String url = in.getHeader("targetUrl", String.class);
-        if (!urlValidator.isValid(url)) {
+
+        // we don't support http
+        if (httpUrlValidator.isValid(url)) {
+            throw new ProtocolException("Insecure protocol is not supported");
```

Comment:
Do we want the stack trace showing in the logs?
Can we make it show on DEBUG mode only?

```
2022-06-01 11:40:20,644 ERROR [org.apa.cam.pro.err.DefaultErrorHandler] (Camel (redhat-splunk-quarkus) thread #4 - KafkaProducer[platform.notifications.fromcamel]) Failed delivery for (MessageId: B22E76E2E5CB385-0000000000000003 on ExchangeId: B22E76E2E5CB385-0000000000000003). Exhausted after delivery attempt: 1 caught: org.apache.http.ProtocolException: Insecure protocol is not supported. Processed by failure processor: FatalFallbackErrorHandler[Channel[direct://secureConnectionFailed]]

Message History (source location and message history is disabled)
---------------------------------------------------------------------------------------------------------------------------------------
Source                                   ID                             Processor                                          Elapsed (ms)
                                         ingress/ingress                from[kafka://platform.notifications.tocamel?broker           15

Stacktrace
---------------------------------------------------------------------------------------------------------------------------------------: org.apache.http.ProtocolException: Insecure protocol is not supported
        at com.redhat.console.notifications.splunkintegration.TargetUrlValidator.process(TargetUrlValidator.java:22)
        at org.apache.camel.support.processor.DelegateSyncProcessor.process(DelegateSyncProcessor.java:65)
        at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$RedeliveryTask.doRun(RedeliveryErrorHandler.java:812)
        at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$RedeliveryTask.run(RedeliveryErrorHandler.java:720)
        at org.apache.camel.impl.engine.DefaultReactiveExecutor$Worker.schedule(DefaultReactiveExecutor.java:193)
        at org.apache.camel.impl.engine.DefaultReactiveExecutor.schedule(DefaultReactiveExecutor.java:59)
        at org.apache.camel.processor.aggregate.AggregateProcessor.lambda$onSubmitCompletion$4(AggregateProcessor.java:893)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
        at java.base/java.lang.Thread.run(Thread.java:831)


2022-06-01 11:40:20,648 WARN  [org.apa.cam.pro.agg.AggregateProcessor] (Camel (redhat-splunk-quarkus) thread #4 - KafkaProducer[platform.notifications.fromcamel]) Error processing aggregated exchange. Exchange[B22E76E2E5CB385-0000000000000003]. Caused by: [org.apache.http.ProtocolException - Insecure protocol is not supported]: org.apache.http.ProtocolException: Insecure protocol is not supported
        at com.redhat.console.notifications.splunkintegration.TargetUrlValidator.process(TargetUrlValidator.java:22)
        at org.apache.camel.support.processor.DelegateSyncProcessor.process(DelegateSyncProcessor.java:65)
        at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$RedeliveryTask.doRun(RedeliveryErrorHandler.java:812)
        at org.apache.camel.processor.errorhandler.RedeliveryErrorHandler$RedeliveryTask.run(RedeliveryErrorHandler.java:720)
        at org.apache.camel.impl.engine.DefaultReactiveExecutor$Worker.schedule(DefaultReactiveExecutor.java:193)
        at org.apache.camel.impl.engine.DefaultReactiveExecutor.schedule(DefaultReactiveExecutor.java:59)
        at org.apache.camel.processor.aggregate.AggregateProcessor.lambda$onSubmitCompletion$4(AggregateProcessor.java:893)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
        at java.base/java.lang.Thread.run(Thread.java:831)
```

Reply:
hmm... it should be only on DEBUG https://github.com/RedHatInsights/eventing-integrations/pull/67/files#diff-ae108e9d88254b537b32e942d1e636818ae87eb672cf551ee713ac1b2ba04702R136